docs/tutorials/tutorials.md

```diff
@@ -0,0 +1,382 @@
+
+**A typical root section for the application definition `NXmpes` outlined**
+
+=== "YAML"
+    ```yaml
+    category: application
+    type: group
+    doc: |
+      This is the most general application definition for multidimensional photoelectron spectroscopy.
+
+      .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+      .. _IUPAC Recommendations 2020: https://doi.org/10.1515/pac-2019-0404
+    symbols:
+      doc: |
+        The symbols used in the schema to specify e.g. dimensions of arrays
+      n_transmission_function: |
+        Number of data points in the transmission function.
+    NXmpes(NXobject):
+    ```
+
+=== "XML"
+
+    ```xml
+    <definition xmlns="http://definition.nexusformat.org/nxdl/3.1"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                category="application"
+                type="group"
+                name="NXmpes"
+                extends="NXobject"
+                xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+      <doc>
+        This is the most general application definition for multidimensional photoelectron spectroscopy.
+        .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+        .. _IUPAC Recommendations 2020: https://doi.org/10.1515/pac-2019-0404
+      </doc>
+      <symbols>
+        <doc>
+          The symbols used in the schema to specify e.g. dimensions of arrays
+        </doc>
+        <symbol name="n_transmission_function">
+          <doc>
+            Number of data points in the transmission function.
+          </doc>
+        </symbol>
+      </symbols>
+    </definition>
+    ```
+### NeXus Group, Field, Attribute, and Link
+#### NeXus Group
+[NeXus groups](https://manual.nexusformat.org/design.html#design-groups), as instances of NeXus base classes, embody the compositional structure of application definitions. Depending on the value of keyword `nameType`, these groups can be initialized dynamically or statically and each approach offers distinct advantages. The keyword `nameType` can hold three district values: `specified`, `any`, and `partial` (for details see the table in `nameType` keyword).
```

Comment:
"can hold three district values: `specified`, `any`, and `partial`" > "can hold one of three distinct values: `specified`, `any`, or `partial`"

Reply:
Addressed!